### ISSUE TITLE
frameless is broken after #367

### ISSUE THREAD

**Issue by aexei** [leanflutter/window_manager#389]
I think since merging [Better fullscreen implementation](https://github.com/leanflutter/window_manager/commit/245f855c169d50c3f13d414c854d27ab3c9e855e) the frameless feature is not working anymore.

Can someone confirm that?

**Comment by damywise** [comment]
Can you try calling `setTitleBarStyle(TitleBarStyle.hidden)` first and then `setAsFrameless` and see if it works now?

If not, can you please provide more context and details to better address the issue:
1. What exactly is not working? Please specify symptoms or behaviors.
2. Which version of Windows are you experiencing the issue on?
3. Please provide steps to reproduce the issue.
4. Could you provide a minimal code snippet that reproduces the issue?

**Comment by damywise** [comment]
If it works, this might be the problem https://github.com/leanflutter/window_manager/blob/36f34eadbe0a7fe0c589052f537ef1bcb502835c/windows/window_manager_plugin.cpp#L115C45-L115C89

**Comment by lijy91** [comment]
@damywise Can you help fix it and submit a new PR?

**Comment by damywise** [comment]
@lijy91 Sorry but I currently don't have access to the Windows machine (yet again 😢 )
Maybe you or @aexei can try moving the line
```
window_manager->title_bar_style_ != "normal"
```
from
https://github.com/leanflutter/window_manager/blob/36f34eadbe0a7fe0c589052f537ef1bcb502835c/windows/window_manager_plugin.cpp#L115
to https://github.com/leanflutter/window_manager/blob/36f34eadbe0a7fe0c589052f537ef1bcb502835c/windows/window_manager_plugin.cpp#L116

It probably won't break the new fullscreen implementation.

**Comment by damywise** [comment]
I finally got access to the Windows machine, need to set it up first tho, should take at most 2 days.

**Comment by damywise** [comment]
@aexei Can you confirm that the linked pull request fixes this?
